### ISSUE TITLE
Brew audit fails on rubocop gem download

### ISSUE THREAD

**Issue by iMichka** [Homebrew/brew#1518]
# Please follow the general troubleshooting steps first:

- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?

### Bug reports:

When doing:
```brew audit --strict --online formula```

I get:
```
==> Installing or updating 'rubocop' gem
ERROR:  Could not find a valid gem 'rubocop' (= 0.45.0), here is why:
          Unable to download data from https://rubygems.org/ - SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed (https://rubygems.org/specs.4.8.gz)
Error: Failed to install/update the 'rubocop' gem.
```

Brew doctor reports no errors; everything is up to date. Brew audit was working some days ago (I pushed the paraview formula on homebrew science lately). So something has changed.

I also tried:
```
brew uninstall --force openssl
brew install openssl
```

And read:
https://github.com/Homebrew/brew/issues/1396

brew config returns:
```
HOMEBREW_VERSION: 1.1.0-136-g34f08a9
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 34f08a9512f86a37af5e95b83f47f5d69c92998c
Last commit: 2 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 25e8d2fd9178a977f050c59e568d187aa3aab097
Core tap last commit: 4 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit haswell
Homebrew Ruby: 2.0.0-p648
Clang: 8.0 build 800
Git: 2.10.2 => /usr/local/bin/git
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.12_2/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 1.8.0_102
macOS: 10.12.1-x86_64
Xcode: 8.1
CLT: 8.1.0.0.1.1476494586
X11: 2.7.11 => /opt/X11
````

```env | grep -i gem``` and ```env | grep -i ruby``` returns nothing.

**Comment by dunn** [comment]
`brew audit` requires rubocop for the style checks, and it's failing to download from rubygems.  Are you behind a proxy?

**Comment by iMichka** [comment]
No, at home. And I changed nothing (at least, not I am aware of). It worked 2 days ago.

```
curl -I  https://rubygems.org/specs.4.8.gz
curl: (60) SSL certificate problem: Invalid certificate chain
More details here: https://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
```

<img width="590" alt="screen shot 2016-11-16 at 00 09 16" src="https://cloud.githubusercontent.com/assets/3406519/20328134/0193dbc4-ab91-11e6-8412-46cc4386b5d3.png">

**Comment by iMichka** [comment]
Yesterday I went to the rubygems.org website with Google Chrome, and there it worked. So I guess Chrome has it's own list of certificates?

Just to let you know that I retried and it now works. I did nothing on my system since I opened the bug. The certificate seems to be valid again. I would have liked to understand what happened ... but the problem does not seem to be on my side but on the rubygems side. 

<img width="589" alt="screen shot 2016-11-16 at 20 30 05" src="https://cloud.githubusercontent.com/assets/3406519/20362637/a173f7cc-ac3c-11e6-86dd-234ba3dc5235.png">
